src/vault.py

```diff
@@ -0,0 +1,39 @@
+from fabric.api import task, local
+from buildercore import project
+import utils
+import sys
+import logging
+LOG = logging.getLogger(__name__)
+
+def vault_addr():
+    defaults, _ = project.raw_project_map()
+    return defaults['aws']['vault']['address']
+
+@task
+def login():
+    cmd = "VAULT_ADDR=%s vault login" % vault_addr()
+    local(cmd)
+
+@task
+def logout():
+    cmd = "rm -f ~/.vault-token"
+    local(cmd)
+
+@task
+def token_lookup(token):
+    cmd = "VAULT_ADDR=%s VAULT_TOKEN=%s vault token lookup" % (vault_addr(), token)
+    local(cmd)
+
+@task
+def token_create():
+    token = utils.get_input('token: ')
+    if not token or not token.strip():
+        print("a token name is required")
+        sys.exit(1)
+    cmd = "VAULT_ADDR=%s vault token create -policy=builder-user -display-name=%s" % (vault_addr(), token)
```

Comment:
`builder-user` is a magic value that eLife's Vault has defined, should probably be a constant. I think we'll later push it up to the configurations, even